packages/flutter/lib/src/painting/binding.dart

```diff
@@ -170,4 +169,4 @@ class _SystemFontsNotifier extends Listenable {
 ///
 /// The image cache is created during startup by the [PaintingBinding]'s
 /// [PaintingBinding.createImageCache] method.
-ImageCache get imageCache => PaintingBinding.instance.imageCache;
+ImageCache? get imageCache => PaintingBinding.instance!.imageCache;
```

Comment:
Same question about the image cache here.

@dnfield (unrelated to this PR): Why do we expose the image cache twice: as part of the binding and as a global? Should one of them be deprecated?

Reply:
That predates my work.  @jonahwilliams might know.  I would be fine with deprecating the global.

Reply:
I don't quite remember. It might have been global before we moved it into a binding

Reply:
Did some digging, looks like it was moved into the binding in https://github.com/flutter/flutter/pull/13409 and the global was kept. I'll prepare a PR to deprecate the global similarly to how we deprecated the global `defaultBinaryMessenger`.